tools/package-tools/build-release.ts

```diff
@@ -26,14 +29,73 @@ export function composeRelease(packageName: string) {
   inlinePackageMetadataFiles(packagePath);
 
   copyFiles(packagePath, '**/*.+(d.ts|metadata.json)', join(releasePath, 'typings'));
-  copyFiles(bundlesDir, `${packageName}.umd?(.min).js?(.map)`, join(releasePath, 'bundles'));
+  copyFiles(bundlesDir, `*.umd?(.min).js?(.map)`, join(releasePath, 'bundles'));
   copyFiles(bundlesDir, `${packageName}?(.es5).js?(.map)`, join(releasePath, '@angular'));
+  copyFiles(join(bundlesDir, packageName), '**', join(releasePath, '@angular', packageName));
   copyFiles(projectDir, 'LICENSE', releasePath);
   copyFiles(packagesDir, 'README.md', releasePath);
   copyFiles(sourcePath, 'package.json', releasePath);
 
   updatePackageVersion(releasePath);
-  createTypingsReexportFile(releasePath, packageName);
-  createMetadataReexportFile(releasePath, packageName);
+  createTypingsReexportFile(releasePath, './typings/index', packageName);
+  createMetadataReexportFile(releasePath, './typings/index', packageName);
+
+  if (options.useSecondaryEntryPoints) {
+    createFilesForSecondaryEntryPoint(packageName, packagePath, releasePath);
+  }
+
   addPureAnnotationsToFile(join(releasePath, '@angular', `${packageName}.es5.js`));
 }
+
+/**
+ * Creates files necessary for a secondary entry-point.
+ * @param packageName The name of the package for which to create entry-point files.
+ * @param packagePath
```

Comment:
This seems to miss a description. In my opinion we don't need those param JSDoc descriptions at all. It feels pretty clear to me.